src/draw.c

```diff
@@ -24,6 +24,8 @@ static SDL_Renderer* mg_pRenderer = NULL;
 static T_HexColor screenColor = {.alphahex = DEF_SCREEN_COLOR_HEX};
 
 static SDL_Texture* mg_playerTexture = NULL;
+
+/*To powinno być przeniesione do innego modułu*/
```

Comment:
Player nodule should be a standalone file, and player hitbox should be mantained in a separate module due to potential complexity of implemented actions regarded to it

Reply:
Will be done in another branch